src/main/java/frc/robot/commands/IntakeCommand.java

```diff
@@ -0,0 +1,75 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Shooter;
+import frc.robot.Constants;
+
+import static frc.robot.Constants.OPERATOR_CONTROLLER;
+
+
+public class IntakeCommand extends Command {
+
+    private final Shooter shooter;
+    private final XboxController controller;
+    private double rightTrigger;
+
+
+    public IntakeCommand(XboxController controller, Shooter shooter) {
+        // each subsystem used by the command must be passed into the
+        // addRequirements() method (which takes a vararg of Subsystem)
+        this.shooter = new Shooter();
+        this.controller = controller;
+        addRequirements();
+    }
+
+    /**
+     * The initial subroutine of a command.  Called once when the command is initially scheduled.
+     */
+    @Override
+    public void initialize() {
+        rightTrigger = controller.getRawAxis(XboxController.Axis.kRightTrigger.value);
+    }
+
+    /**
+     * The main body of a command.  Called repeatedly while the command is scheduled.
+     * (That is, it is called repeatedly until {@link #isFinished()}) returns true.)
+     */
+    @Override
+    public void execute() {
+        this.shooter.runShooter(-(rightTrigger),-(rightTrigger));
```

Comment:
Clean this up a bit. you can just do -rightTrigger instead. Also same issues as the intake command